.qovery.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Not sure to understand why you add this file

We don't use Qovery on this repo or the Docusaurus website, so we don't really want to add a config file for a provider that we don't use (or it would open the door to an infinite number of config files at the root of our repo: we probably don't want that)

Reply:
Sorry for this git command mistake.